source/Halibut.Tests/Support/BackwardsCompatibility/PreviousVersions.cs

```diff
@@ -16,8 +16,8 @@ public static class PreviousVersions
 #if NETFRAMEWORK
         public static readonly HalibutVersions v4_4_8 = new(new Version("4.4.8"), pollingOverWebSocketServiceVersion: v6_0_658_WebSocket_Stability_Fixes);
 #else
-        // A net6.0 client causes an older halibut service without buffer over-read changes to be prone to hanging or erroring when reading the stream.
-        // This instability means we cannot really have automated tests for net6.0 and older these older Halibut Services
+        // A net8.0 client causes an older halibut service without buffer over-read changes to be prone to hanging or erroring when reading the stream.
+        // This instability means we cannot really have automated tests for net8.0 and older these older Halibut Services
```

Comment:
As above. Although these are tests, the original comment was probably more accurate.

Reply:
I guess since we are only testing net8.0 client now (against older services), then the update is ok.
The second comment line doesn't make sense and needs fixing anyway:
"...and older these older..."